libraries/bezier-rs/src/subpath/solvers.rs

```diff
@@ -31,8 +34,10 @@ impl Subpath {
 
 	/// Calculates the intersection points the subpath has with a given line and returns a list of parameteric `t`-values.
 	/// This function expects the following:
-	/// - other: a [Bezier] curve to check intersections against
-	/// - error: an optional f64 value to provide an error bound
+	/// - `other`: a [Bezier] curve to check intersections against
+	/// - `error`: an optional f64 value to provide an error bound
+	/// - minimum_seperation: the minimum difference 2 adjacent `t`-values must have, when comparing adjacent `t`-values, if 2 `t`-values do not satisfy the
```

Comment:
itty bitty nit: can you put ticks around this argument like the two above? Also, you could probably split this sentence at the if so it flows a bit nicer (here and below)